app/api/helpers/utilities.py

```diff
@@ -120,3 +120,13 @@ def update_state(task_handle, state, result=None):
 
 class EmptyObject(object):
     pass
+
+
+def pg_conform_search(search):
+    """
+    Makes `search` a valid term for Postgres full-text search
+    See the documentation for more information:
+    https://www.postgresql.org/docs/9.5/static/textsearch-intro.html#TEXTSEARCH-MATCHING
+    """
+    search = search.replace(' ', '&')
+    return search
```

Comment:
May be you can move this to `db.py` helper since it is related to the database ?